src/Test.UnitTests.Sarif/Core/WebResponseTests.cs

```diff
@@ -120,20 +118,19 @@ public void WebResponse_Parse_ThrowsIfResponseDoesNotEndWithABlankLine()
         }
 
         [Theory]
-        [InlineData("", false, null, null, null, -1, null, -1, "status line is empty")]
```

Comment:
>null [](start = 31, length = 4)

Remove the first `null` argument (corresponding to the now-removed `httpVersion` property) in each data line.